corehq/apps/accounting/enterprise.py

```diff
@@ -63,30 +76,85 @@ def rows_for_domain(self, domain):
 
 
 class EnterpriseWebUserReport(EnterpriseReport):
-    def __init__(self):
-        super(EnterpriseWebUserReport, self).__init__()
+    def __init__(self, couch_user):
+        super(EnterpriseWebUserReport, self).__init__(couch_user)
 
     @property
     def headers(self):
         headers = super(EnterpriseWebUserReport, self).headers
         return ['Name', 'Email Address', 'Role', 'Last Login'] + headers
 
+    def rows_for_domain(self, domain):
+        rows = []
+        for user in get_all_user_rows(domain.name, include_web_users=True, include_mobile_users=False,
+                                      include_inactive=False, include_docs=True):
+            user = WebUser.wrap(user['doc'])
```

Comment:
I've recently gotten myself into the habit of always using `CouchUser.wrap_correctly`, the reason being that the user classes unfortunately have lazy migrations with side effects on wrap, which are different for both user class types. So on the off chance that this result is mismatched some unexpected behavior might happen during the lazy migration, and using wrap_correctly prevents that.

Reply:
I didn't know about `wrap_correctly`, thanks